testing/lxd/APKBUILD

```diff
@@ -1,7 +1,8 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Contributor: Adrian Pistol <vifino@tty.sh>
```

Comment:
I think the "Contributor" field is for who contributed the package to Alpine

Reply:
Not for those who contributed to the package?

Reply:
Yes, who created the abuild or significantly improved. Package bump doesn't fit this category.

Reply:
Ah. I did contribute more than just this to the LXD abuild, fixing it up, but I'll recommit without that line.